support-frontend/assets/components/countryGroupSwitcher/countryGroupSwitcher.tsx

```diff
@@ -11,20 +14,37 @@ import { currencies } from 'helpers/internationalisation/currency';
 import { sendTrackingEventsOnClick } from 'helpers/productPrice/subscriptions';
 import type { SubscriptionProduct } from 'helpers/productPrice/subscriptions';
 import type { Option } from 'helpers/types/option';
-import './countryGroupSwitcher.scss';
-import moduleStyles from './countryGroupSwitcher.module.scss';
 
-const styles = moduleStyles as { button: string };
+const countryGroupSwitcherButton = css`
+	appearance: none;
+	border: 0;
+	padding: 0;
+	color: inherit;
+	cursor: pointer;
+	color: ${palette.neutral[100]};
+	${textSans17};
```

Comment:
The previous version was size 16, this doesn't exist in source so I went with 17px